### ISSUE TITLE
Error during first usage (try to commit sketch file)

### ISSUE THREAD

**Issue by thomaspannek** [mathieudutour/git-sketch-plugin#213]
### How did it happen?
1. Installed Plugin for the first Time Sketch Version 55.2
2. init Repo over advanced menu
3. add sketch file
4. commit file

### Error log

```
{
	"nativeException": {},
	"line": 195,
	"column": 37,
	"sourceURL": "/Users/tjuice/Library/Application Support/com.bohemiancoding.sketch3/Plugins/Git.sketchplugin/Contents/Sketch/Commit.js"
}
```

**Comment by MarcSteven** [comment]
@thomaspannek  Perhaps it had been paused.

**Comment by mathieudutour** [comment]
fixed in 0.12
